src/APIClient/GPURequestor.cpp

```diff
@@ -0,0 +1,25 @@
+#include "APIClient/GPURequestor.hpp"
+
+int APIClient::GPURequestor::putVulkanInfo(nlohmann::json vulkanInfosJson) {
+    std::cout << vulkanInfosJson.dump() << std::endl;
```

Comment:
Do you really want to display the JSON here? I don't really care if you display or not the JSON, but it's probably better to do it in another place.